gemini-resin-archetype/src/main/resources/archetype-resources/src/main/java/com/techempower/handlers/HomeHandler.java

```diff
@@ -4,19 +4,34 @@
 import com.techempower.gemini.path.*;
 import com.techempower.gemini.path.annotation.*;
 
-public class HomeHandler extends MethodUriHandler<Context>
+public class HomeHandler extends MethodSegmentHandler<Context>
 {
 
   public HomeHandler(GeminiApplication app)
   {
     super(app);
   }
-  
-  @Path
+
+  //@Path
+  @PathRoot
   @Get
   public boolean home()
   {
     return mustache("home");
   }
+
+  //@Path("example")
+  @PathSegment("example")
+  @Get
+  public boolean example()
+  {
+    return mustache("example");
+  }
   
+  @PathDefault
+  public boolean notFound()
+  {
+    //mustache("common/error");
+    return notFound("Not found");
```

Comment:
Is the use of `MethodSegmentHandler` simply for the `notFound` method?

Reply:
My apologies for the commented-out bits of code. I just pushed a fix to remove those.

I switched to MethodUriHandler because it fits better with the server-side rendering we're demonstrating here. Happy to use MethodUriHandler instead if you can show me how it can show a server-side rendered 404 page? I'm probably missing something obvious.